Makefile

```diff
@@ -10,7 +10,7 @@ run-python:
 
 format:
 	@poetry run ruff format .
-	@poetry run ruff .
+	@poetry run ruff . --fix
```

Comment:
@cuducos adicionei essa flag para o Ruff já tentar corrigir automaticamente o erro se for algo "fixable"

Reply:
Opa, boa!